src/property-filter/index.tsx

```diff
@@ -235,12 +234,6 @@ const PropertyFilter = React.forwardRef(
       addToken(newToken);
       setFilteringText('');
     };
-    const ignoreKeyDown = useRef<boolean>(false);
```

Comment:
This code should not exist in the first place. It was added to get around the missing `onSelect` event, which is fixed now